File: Group6System-main/Group6System-main/BlazorClient/BlazorClient/Models/AdminValidation.cs
using System.Text.Json.Serialization;

namespace BlazorClient.Models
{
    public class AdminValidation
    {
        [JsonPropertyName("customeraccount")]
        public Account CustomerAccount { get; set; }
        [JsonPropertyName("customer")]
        public Customer Customer { get; set; }
    }
}

File: Group6System-main/Group6System-main/ApplicationTier/src/main/java/com/example/applicationtier/models/AdminValidation.java
package com.example.applicationtier.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonSetter;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Component
public class AdminValidation {
    @JsonProperty("customeraccount")
    private Account account;
    @JsonProperty("customer")
    private Customer customer;
}
